TI-reports/2024/2024-Q4-Alpha-Omega.md

```diff
@@ -0,0 +1,94 @@
+
+* **Alpha-Omega Roundtable**: We held a roundtable for Alpha-Omega partners and friends at the Open Source Summit
+  in Vienna. We had a lively discussion on many topics, including end-of-life transitions for open source projects,
+  and how to better market our successes to drive awareness and further investment.
+  
+* **AI Library Reviews**: The security audit of 25 of the most popular AI libraries has wrapped up, and we're currently
+  discussing next steps, and will share details when we're able to.
+
+* **Engagement**: We recently expanded our engagement with Node.js to cover OpenJS, started an engagement with
+  Trail of Bits of improve PyPI's project-level lifecycle functionality, and kicked off a new type of engagement ("Beach Cleaning")
+  with Apache Airflow and its 700+ dependencies. We also renewed funding for the ClangBuiltLinux project, and kicked
+  off work to bring Content Security Policy (CSP) to Jenkins, which will better protect Jenkins infrastructure from vulnerable plugins.
+  Details about each of these engagement are available in our GitHub repository.
+
+## Objectives & Key Results
+
+All of our OKRs are on-track except for KR 2.1.
+
+|Key Result|Status|
+|-|-|
+|**O1: Catalyze trustworthy and secure software, runtimes, and infrastructure for all the major open source ecosystems through staffing**||
+|KR 1.1: Fund security improvements and initiatives for at least ten critical open source organizations by the end of 2024. |On target|
+|KR 1.2: For each engagement, confirm progress toward improved security outcomes, evidenced through initial and/or follow-on assessments, monthly reporting, and periodic check-ins.|On target|
+|KR 1.3: Drive the organizations we work with to obtain security funding from at least one organization other than Alpha-Omega, targeting 33% by the end of 2024.|On target|
+|KR 1.4: Organize quarterly roundtables for at least 5 major ecosystems  to share information, build connections, and collaborate, resulting in at least one new project or joint publication started in 2024.|On target|
+|**O2: The top 10,000 open source projects are free of critical security vulnerabilities**||
+|KR 2.1: Drive adoption of key security processes, including static analysis, credential scanning, the use of private vulnerability disclosures, structured metadata (Security Insights) and the use of multi-factor authentication by maintainers of 500 critical projects from the top 10,000 by the end of 2024.|Not Started|
+|KR 2.2: Independently scan, triage, and notify maintainers when critical vulnerabilities are found in 2,000 projects, chosen from the top 10,000 by the end of June 2024, with emphasis on clearing a "section of the beach" by focusing on the top PyPI packages.|On target|
+|KR 2.3: Publish in a machine readable format the attestations for all packages from 2.2 that returned no vulnerabilities and those that found vulnerabilities which were subsequently fixed and verified.|On target|
```

Comment:
This is more out of curiosity: What's the format you're using for these attestations? Is it in-toto, or some other format?

Reply:
We're experimenting with in-toto attestations, OpenRefactory has been storing them in [OpenRefactory-Inc/attestations](https://github.com/OpenRefactory-Inc/attestations); for example, https://github.com/OpenRefactory-Inc/attestations/blob/master/addict/2.3.0/2024-07-04/attestation.json